loom/script/native/lsNativeInterface.cpp

```diff
@@ -204,7 +204,8 @@ void NativeInterface::resolveScriptType(Type *type)
         }
     }
 
-    lmAssert(0, "Unable to resolve script type %s", type->getFullName().c_str());
+    type->setMissing("resolveScriptType");
```

Comment:
OK. Well maybe it's worth just documenting that the base method does nothing and exists to let subclasses override?